test/hoss/version.txt

```diff
@@ -1 +1 @@
-0.1.4
+0.1.5
```

Comment:
Wasn't really sure on the best type of version to bump. I don't think it's a major version bump, but could be convinced to make this a minor version bump.

Reply:
No strong opinions, so this is fine with me.